src/MemoryRouter.test.tsx

```diff
@@ -62,6 +68,19 @@ describe("MemoryRouter", () => {
           });
         });
 
+        it("should trigger only hashEvents on a hash change event", async () => {
+          routeChangeStart.mockClear();
+          routeChangeComplete.mockClear();
+          hashChangeStart.mockClear();
+          hashChangeComplete.mockClear();
+
```

Comment:
There's already a `beforeEach` that calls `jest.clearAllMocks()`
```suggestion
```

Reply:
Okay - I'll fix that